inst/script.R

```diff
@@ -1,10 +1,16 @@
 # see `help(run_script, package = 'touchstone')` on how to run this
 # interactively
 
-touchstone::refs_install() # installs branches to benchmark
+# TODO Add directories you want to be available in this file or during the
```

Comment:
I think all TODO things should be marked optional and commented out so the user has a minimal script where he has to change only this line `random_test = yourpkg::f(), #<- TODO put the call you want to benchmark here`.  Like this:
```r
# TODO OPTIONAL Add directories you want to be available in this file or during the
# benchmarks.
# touchstone::pin_head_asssets("some/dir")
```

Can you edit all other places where this is relevant in `inst/script.R`?